mods/ra/maps/top-o-the-world/rules.yaml

```diff
@@ -61,13 +61,15 @@ powerproxy.spyplane:
 		EndChargeSpeechNotification: SpyPlaneReady
 		CameraActor: camera.spyplane
 		CameraRemoveDelay: 150
-		UnitType: u2
 		QuantizedFacings: 8
 		DisplayBeacon: true
 		BeaconPoster: camicon
 		ArrowSequence: arrow
 		ClockSequence: clock
 		CircleSequence: circles
+		Squad@override:
```

Comment:
What is this overriding?

Reply:
That is a good question and I don't have an answer. Removed the suffix as it seems unnecessary.